PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -556,9 +556,9 @@ def miniAOD_customizeCommon(process):
     from Configuration.Eras.Modifier_run2_L1prefiring_cff import run2_L1prefiring
     from Configuration.Eras.Modifier_stage1L1Trigger_cff import stage1L1Trigger
     from Configuration.Eras.Modifier_stage2L1Trigger_2017_cff import stage2L1Trigger_2017
-    process.load("PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff")
-    stage1L1Trigger.toModify(process.prefiringweight, DataEra = "2016BtoH")
-    stage2L1Trigger_2017.toModify(process.prefiringweight, DataEra = "2017BtoF")
+    process.load("PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff")
+    stage1L1Trigger.toModify(process.prefiringweight, DataEraECAL = "2016BtoH", DoMuons = cms.bool(False) )
+    stage2L1Trigger_2017.toModify(process.prefiringweight, DataEraECAL = "2017BtoF", DoMuons = cms.bool(False))
```

Comment:
In order to get the original weights, I think you need to also switch off the protection against jets with high muon energy fraction. This can be achieved by setting: 
`JetMaxMuonFraction = -1`
both for 2016 and 2017

Reply:
Ok, done